scripts/boot-master/copy_cluster_skel.sh

```diff
@@ -12,7 +12,9 @@ source /tmp/icp-bootmaster-scripts/functions.sh
 parse_icpversion ${1}
 echo "registry=${registry:-not specified} org=$org repo=$repo tag=$tag"
 
-
+# Ensure that /opt/ibm is present and copy default data directory
+sudo mkdir -p /opt/ibm
+sudo chown $(whoami):$(whoami) -R /opt/ibm
```

Comment:
Is it always the case that users are added to a group with the same name?
I think it's normally the case, but I don't know how safe an assumption it is with different distributions that may be used?

Reply:
I thought that it was the case. We can also add them to the first group as reported by `groups`.